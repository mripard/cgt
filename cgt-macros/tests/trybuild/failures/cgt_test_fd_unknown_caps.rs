#[cgt_macros::cgt_test_with_fd(capabilities=[Unknown])]
fn test(_: std::os::fd::BorrowedFd<'_>) -> Result<(), cgt_core::TestError> {
    Ok(())
}

fn main() {}