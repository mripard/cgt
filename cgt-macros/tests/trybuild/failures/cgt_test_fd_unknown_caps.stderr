error[E0425]: cannot find value `Unknown` in this scope
 --> tests/trybuild/failures/cgt_test_fd_unknown_caps.rs:1:46
  |
1 | #[cgt_macros::cgt_test_with_fd(capabilities=[Unknown])]
  |                                              ^^^^^^^ not found in this scope
